index.rst

```diff
@@ -1,61 +1,263 @@
+The technology defines a powerfull real-time message system protocol with high inter-operability between platforms and programming languanges.
+In fact, DDS has most of the important features we recognize as crucial for the Rubin-OCS.
+Some of the most important ones are, for example:
+
+-  Real-time message transfer capabilities.
+
+   DDS is a brokerless messaging system with small overhead.
+   As such, it is capable of high-throughput and low-latency data transfer.
+   For example, in our `internal benchmarks`_, DDS reached transfer rates of the order of >16kHz with millisecond latency.
+   This is certainly way beyond the requirements of our system, which are mostly constrained by the throughput of M1M3 (REQ?) and the throughput/latency required for tracking (LTS-TCS-PTG-0008 and LTS-TCS-PTG-0001 in :lts:`583` specify lead-times between 50-70ms with standard deviation of 3ms for tracking demands).
+
+-  Durability service.
+
+   In messaging systems, "durability" refer to the capability of a system to store published data and serve it to components that join the system afterwards.
+   This service is crucial for a distributed system like Rubin-OCS as it guarantees that components comming online at any time are able to determine the state of the system by accessing previously published information.
+
+   A surprising number of systems do not provide any type of durability service, especially those that are deemed "real-time".
+
+   This mostly boils down from the fact that most real-time capable systems are brokerless (like DDS).
+   Nevertheless, in order to provide a durability service, a system must have some kind of broker, that can store published messages and distribute them when needed.
+
+   DDS provides a rather elegant solution to this problem.
+   Basically, each independent node can be configure to act as a broker for durability service.
+   One of those systems is elected as the "master" node, which will be in charge of actually distributing the data.
+   If the master node falls over, some other node is elected to take its place.
+
+   Nevertheless, as we have demonstrated in our efforts to stabilize the system (:tstn:`023`), this can have a huge impact in the system performance and adds considerable complexity in configuring the system.
+
+-  The Quality of Service (QoS) dictates how messages are delivered under different network scenarios.
+
+   DDS has an extremely rich QoS system with several configuration parameters.
+   Nevertheless, while this might sound like a desirable feature at a first glance, it has some serious implications.
+   To begin with, a large number of configuration parameters also means higher complexity, which makes it harder to predict the system behaviour under unexpected conditions.
+   We have encoutered inumerous unexpected behaviour that were later linked to a certain unexpected bahaviour cause by a default setting.
```

Comment:
"inumerous" and "behaviors". Also please don't duplicate "unexpected".

Consider something along the lines of "We have encountered many problems that were traced to unexpected behavior caused by a QoS setting"